Tests/CKAN/Relationships/RelationshipResolver.cs

```diff
@@ -47,12 +48,22 @@ public void Constructor_WithConflictingModules_Throws()
             list.Add(mod_a.identifier);
             list.Add(mod_b.identifier);
             AddToRegistry(mod_a, mod_b);
-
+            
             Assert.Throws<InconsistentKraken>(() => new RelationshipResolver(
                 list,
                 options,
                 registry,
                 null));
+
+
+            options.procede_with_inconsistencies = true;
+            var resolver = new RelationshipResolver(list, options, registry, null);
+            var a = resolver.ConflictList.First();
+            var b = resolver.ConflictList.First();
```

Comment:
- [ ] I might be missing something here, but we don't seem to use `a` or `b` at all...